search-app/docs/api.md

```diff
@@ -4099,8 +4110,38 @@ Content-Length: 292
   } ]
 }
 ```
+__Example Response With Associations__
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=UTF-8
+Content-Length: 512
+
+{
+  "hits" : 1,
+  "took" : 41,
+  "items" : [ {
+    "concept_id" : "V1200000022-PROV1",
+    "revision_id" : 2,
+    "provider_id" : "PROV1",
+    "native_id" : "sample-variable",
+    "name" : "methaneVar",
+    "long_name" : "Total Methane",
+    "associations" : {
```

Comment:
With the documentation, it makes this observation clearer: For JSON result format that does not originally have the `associations` field, do we want to make the result more compact with `details` in the associations field, or do we want to make the `associations` and `association_details` fields separate just to be the same as old JSON result format which is not clean? I am OK with your choice here, but I just want to point this out that you made this choice (perhaps unintentionally).